test/unit/user_test.rb

```diff
@@ -206,10 +206,6 @@ class UserTest < ActiveSupport::TestCase
       @rubygems.first.versions.first.update! indexed: false
       assert_equal 2, @user.rubygems_downloaded.count
     end
-
-    should "total their number of pushed rubygems" do
-      assert_equal @user.total_rubygems_count, 3
-    end
```

Comment:
This test should not be deleted. It should be updated to ensure that yanked gems are not counted.